third-party/eazy/v1/openapi.yaml

```diff
@@ -2333,6 +2333,8 @@ components:
           description: Type of the contact
           type: string
           example: 'INDIVIDUAL'
+    note:
+      description: TODO
```

Comment:
```suggestion
      description: 'Array of notes about Contact'
      type: array
      items:
          $ref: '#/components/responses/noteResponse'
```

Based on:
```
 "notes": [
        {
            "body": "Return order received",
            "createdAt": "2020-03-11T19:53:02.000000Z",
            "createdBy": "john@mycompany.com",
            "id": "280cbbe0-c08a-4606-bf8d-a30242c4eaa4",
            "isReadOnly": false,
            "updatedAt": null,
            "referenceId": null
        }
    ]
    
```

Reply:
Again, just a suggestion :)

Reply:
I don't think so. `contactResponse` contains a `notes` property which is an array of `note`. After your change, `note` becomes an array of `noteResponse`. Thus, after your change, the `notes` property of `contactResponse` becomes an array of arrays of objects:
```json
{
  "notes": [
    [
      {
        "body": "Return order received",
        "createdAt": "2020-03-11T19:53:02.000000Z",
        "createdBy": "john@mycompany.com",
        "id": "280cbbe0-c08a-4606-bf8d-a30242c4eaa4",
        "isPinned": false,
        "isReadOnly": false,
        "updatedAt": "2020-04-11T19:53:02.000000Z",
        "referenceId": "CUSTOMER_NUMBER"
      }
    ]
  ],
  ...
}
```

I don't think that is correct.

Reply:
U are right, I miss-viewed. Please, fix the TO-DO then. Ty

Reply:
@ridlees, and how can I fix it? I have no idea what `note` is. The `TODO` there is just a placeholder to me so that I can remember what should be fixed. Your suggested `Array of notes about Contact` probably does not work :arrow_up: . So if you know what it is, please tell me. Otherwise, if you agree, I am just going to close this PR (instead of merging) and to ask Peter in [the upstream PR](https://github.com/tyntec/api-collection/pull/103).

Reply:
That is what I mean by fixing it (to ask in upstream). Apologies for the unclear instruction.